jupyterlab_bigquery/jupyterlab_bigquery/details_handler/service.py

```diff
@@ -161,7 +163,9 @@ class BigQueryService:
 
   _instance = None
 
-  def __init__(self, client=bigquery.Client()):
+  def __init__(self,
+               client=bigquery.Client(client_info=ClientInfo(
+                   user_agent='jupyterlab_bigquery/{}'.format(VERSION)))):
```

Comment:
For all of these can you just set it to `'jupyterlab_gcpextension'`

Reply:
Per our discussion set `user_agent` to `'jupyterlab_gcpextension/jupyterlab_bigquery-{}'.format(VERSION)`